docs/tests/e2e-web.md

```diff
@@ -55,13 +53,43 @@ It is possible to run tests with image snapshots to test for visual regressions.
 When you need to update image snapshots you have 2 options:
 - use CI job. This will generate new snapshots in artifacts together with a handy script that updates your snapshots locally. Check the log output. 
 - use `./docker/docker-suite-snapshots.sh`. This does the same as `./docker/docker-suite-test.sh`, the only difference is it won't fail on non-matching snapshots but generate new snapshots.
-# Old notes (to be removed) 
-
-## Notes on bridge (trezord)
+# Notes
+
+## Running e2e tests locally macOS
+1. Follow first three steps in macOS category at [trezor-user-env](https://github.com/trezor/trezord-go/tree/cypress) repository.
```

Comment:
Could you elaborate what three steps you have in mind?

Reply:
Oh or maybe just the link is incorrect and it should lead to master? Because I am 90% sure we do not need Martin's `cypress` branch anymore.

Reply:
:+1: one small additional suggestion:

1. Install prerequisites and follow first three steps in [trezor-user-env](https://github.com/trezor/trezor-user-env#macos) repository.

Or actaully we should even copy & paste it here? Because the README can change in trezor-user-env. Let me know if I am bikesheding too much @tomasklim

Reply:
Good point, I will copy & paste it. The steps required will probably not change any time soon but the trezor-user-env repository can be changed or moved.

Reply:
done